spring-data-microservice-transaction-sample/README.md

```diff
@@ -0,0 +1,552 @@
+| 2       | Orange | 2000  |
+| 3       | Grape  | 2500  |
+| 4       | Mango  | 5000  |
+| 5       | Melon  | 3000  |
+
+## Run the sample application
+
+Let's start with getting information about the customer whose ID is `1`:
+
+```shell
+$ ./gradlew :client:run --args="GetCustomerInfo 1"
+...
+{"id": 1,"name": "Yamada Taro","credit_limit": 10000}
+...
+```
+
+At this time, `credit_total` isn't shown, which means the current value of `credit_total` is `0`.
+
+Then, place an order for three apples and two oranges by using customer ID `1`.
+Note that the order format is `<Item ID>:<Count>,<Item ID>:<Count>,...`:
+
+```shell
+$ ./gradlew :client:run --args="PlaceOrder 1 1:3,2:2"
+...
+{"order_id": "415a453b-cfee-4c48-b8f6-d103d3e10bdb"}
+...
+```
+
+You can see that running this command shows the order ID.
+
+Let's check the details of the order by using the order ID:
+
+```shell
+$ ./gradlew :client:run --args="GetOrder 415a453b-cfee-4c48-b8f6-d103d3e10bdb"
+...
+{"order": {"order_id": "415a453b-cfee-4c48-b8f6-d103d3e10bdb","timestamp": 1686555272435,"customer_id": 1,"customer_name": "Yamada Taro","statement": [{"item_id": 1,"item_name": "Apple","price": 1000,"count": $
+,"total": 3000},{"item_id": 2,"item_name": "Orange","price": 2000,"count": 2,"total": 4000}],"total": 7000}}
+...
+```
+
+Then, let's place another order and get the order history of customer ID `1`:
+
+```shell
+$ ./gradlew :client:run --args="PlaceOrder 1 5:1"
+...
+{"order_id": "069be075-98f7-428c-b2e0-6820693fc41b"}
+...
+$ ./gradlew :client:run --args="GetOrders 1"
+...
+{"order": [{"order_id": "0b10db66-faa6-4323-8a7a-474e8534a7ee","timestamp": 1631605501485,"customer_id": 1,"customer_name": "Yamada Taro","statement": [{"item_id": 5,"item_name": "Melon","price": 3000,"count": 1,"total": 3000}],"total": 3000},{"order_id": "4ccdb21c-ac03-4b48-bcb7-cad57eac1e79","timestamp": 1631605253126,"customer_id": 1,"customer_name": "Yamada Taro","statement": [{"item_id": 1,"item_name": "Apple","price": 1000,"count": 3,"total": 3000},{"item_id": 2,"item_name": "Orange","price": 2000,"count": 2,"total": 4000}],"total": 7000}]}
+
+{"order": [{"order_id": "069be075-98f7-428c-b2e0-6820693fc41b","timestamp": 1686555279366,"customer_id": 1,"customer_name": "Yamada Taro","statement": [{"item_id": 5,"item_name": "Melon","price": 3000,"count": 1,"total": 3000}],"total": 3000},{"order_id": "415a453b-cfee-4c48-b8f6-d103d3e10bdb","timestamp": 1686555272435,"customer_id": 1,"customer_name": "Yamada Taro","statement": [{"item_id": 1,"item_name": "Apple","price": 1000,"count": 3,"total": 3000},{"item_id": 2,"item_name": "Orange","price": 2000,"count": 2,"total": 4000}],"total": 7000}]}
+...
+```
+
+This order history is shown in descending order by timestamp.
+
+The customer's current `credit_total` is `10000`.
+Since the customer has now reached their `credit_limit`, which was shown when retrieving their information, they cannot place anymore orders.
+
+TODO: Check the stack trace
```

Comment:
TODO is remaining.

Reply:
Thanks. Fixed it in https://github.com/scalar-labs/scalardb-samples/pull/37/commits/2c071e5e1bb859c157b910a9a97baa81a5d10dd6